website/docs/spec/facets/dataset-facets/database_hierarchy.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 12
+---
+
+# Database Hierarchy Dataset Facet
+
+The facet contains information about the hierarchy of objects within database.
+
+Fields description:
+- `objectType`: Type of the object within the hierarchy (e.g. `TABLE`, `VIEW`, `TOPIC`).
+- `hierarchy`: Database hierarchy levels (e.g. `DATABASE` -> `SCHEMA` -> `TABLE`), from highest to lowest level. Array of objects:
+  - `type`: Hierarchy level type (e.g. `DATABASE`, `SCHEMA`, `TABLE`).
+  - `name`: Hierarchy level object name (e.g. `my_db`, `my_schema`, `my_table`).
+
+`objectType` is required field. `hierarchy` is not, because some databases have no hierarchy concept (e.g. Kafka topics).
```

Comment:
Why include the facet then? For `objectType` alone we can have https://openlineage.io/spec/facets/1-0-0/DatasetTypeDatasetFacet.json#/$defs/DatasetTypeDatasetFacet

Reply:
Done